cpp/cuproj/README.md

```diff
@@ -1,7 +1,43 @@
 # <div align="left"><img src="https://rapids.ai/assets/images/rapids_logo.png" width="90px"/>&nbsp;cuProj: GPU-Accelerated Coordinate Projection</div>
```

Comment:
I recently wrote a fair amount in the readme for cuProj that you might want to merge from PR #1217.

Reply:
Will do!